sdk/storage/azure-storage-queue/apireview/azure/storage/queue/_message_encoding.pyi

```diff
@@ -0,0 +1,48 @@
+# Stubs for azure.storage.queue._message_encoding (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._shared.encryption import decrypt_queue_message, encrypt_queue_message
+from typing import Any
+
+class MessageEncodePolicy:
+    require_encryption: bool = ...
+    key_encryption_key: Any = ...
+    resolver: Any = ...
+    def __init__(self) -> None: ...
+    def __call__(self, content: Any): ...
+    def configure(self, require_encryption: Any, key_encryption_key: Any, resolver: Any) -> None: ...
+    def encode(self, content: Any) -> None: ...
+
+class MessageDecodePolicy:
+    require_encryption: bool = ...
+    key_encryption_key: Any = ...
+    resolver: Any = ...
```

Comment:
Thoughts on providing a default KeyResolver? It might be used for client side encryption too.